lahja/asyncio/endpoint.py

```diff
@@ -718,13 +599,14 @@ def stop(self) -> None:
         where this event should be broadcasted to. By default, events are broadcasted across
         all connected endpoints with their consuming call sites.
         """
-        await self._broadcast(item, config, None)
+        await self._broadcast(item, config, None, None)
 
     async def _broadcast(
         self,
         item: BaseEvent,
         config: Optional[BroadcastConfig],
         id: Optional[RequestID],
+        remotes_for_broadcast: Optional[Tuple[RemoteEndpointAPI, ...]],
```

Comment:
Unless I'm missing something, this last parameter of `_broadcast` is only `not None` when the last parameter (`remotes_for_broadcast`) of `_request` is `not None`, but the only place where `_request` is called always passes in `None`. 

Is this leftover code, or maybe you plan on using it in a future PR?

Reply:
Yeah, I realized that a bunch of this machinery can now be removed since we no longer need to use these internals.